__mocks__/handlers.ts

```diff
@@ -0,0 +1,9 @@
+import taskHandlers from "./handlers/tasks.handler";
+import selfHandler from "./handlers/self.handler";
+import idleUserHandler from './handlers/idle-users.handler';
+import tagsHandler from './handlers/tags.handler';
+import levelsHandler from './handlers/levels.handler'
+
+const handlers = [...taskHandlers, ...selfHandler, ...idleUserHandler, ...tagsHandler, ...levelsHandler];
```

Comment:
```ts
function combineHandlers(...handlers: handler[]){
return handlers.reduce((acc, handler) => [...acc,...handler], []);
}

`Suggestion:` Check the above snippet to use instead of manually destructuring and using.
```